AcquisitionInfo/explainer.md

```diff
@@ -0,0 +1,296 @@
+documentation and are not final.
+
+```js
+details = {
+  installSource: "Google Play Store",
+  installReferrer: "utm_source=google-play&utm_medium=organic",
+  referrerClickTimestampSeconds: 1623214800,
+  installBeginTimestampSeconds: 1623214810,
+  installBeginReferrer: "utm_source=google-play&utm_medium=referral&utm_campaign=campaign-123",
+  installVersion: "1.0.0",
+  googlePlayInstant: false
+}
+```
+
+```js
+details = {
+  installSource: "Apple App Store",
+  attribution: "true",
+  orgId: "40669820",
+  campaignId: "542370539",
+  conversionType: "Download",
+  adGroupId: "542317095",
+  countryOrRegion: "US",
+  keywordId: "87675432",
+  adId: "542317136"
+}
+```
+
+In the event that a web application has referral information that has expired for that specific
+application, or in any other case where acquisition info is missing for the web application,
+the API would return empty values for the associated properties in the dictionary. This is consistent with
+existing behavior in store information retrieval API for native applications.
+
+```js
+details = {
+  installSource: "Microsoft Store",
+  campaignId: ""
+}
+```
+
+## Attribution on browser installs
+
+While attribution is currently not being tracked in the same capacity as Store installations for browser-initiated
+installations of web applications, the Acquisition Info API could enable this sort of attribution capability.
+
+### Same-domain installation
+
+Currently the main method of installing a PWA through the browser is navigating to the desired web application and
+initiating a same-domain installation. It may be the case that the intial navigation was due to an ad click.
+By having a standardized dictionary property (`attributionId`) that the UA is responsible for capturing to keep track of ad attribution,
+it becomes possible for the Acquistion Info API to return attribution information for browser-installed
+web applications as well. This `attributionId` dictionary property could be captured by the UA through a query parameter such
+as `__a_id` in the [HTTP header `Referer`][HTTP Referer] field.
+> This solution would require that the referrer policy is not specified to block `referer` information requests.
+
+#### Example same-domain use case
+
+1. User clicks on a Bing ads campaign on `bar.com` that navigates to `foo.com`.
+The HTTP `Referer` passed to `foo.com` is `bar.com?__a_id=bingAdsAug2023`.
+
+2. User installs the `foo.com` PWA from `foo.com`.
+
+3. UA captures the attribution information at the time of installation.
+
+4. Calling the Acquisition Info API `getDetails()` for `foo.com` PWA produces the following payload.
+
+> Proposed dictionary properties are not final and subject to change based on feedback.
+
+```js
+details = {
+  installSource: "foo.com",
+  attributionId: "bingAdsAug2023",
+  httpReferrer: "bar.com?__a_id=bingAdsAug2023",
```

Comment:
This one would just be `bar.com` (unless the user came to `bar.com` with addition GET parameters in that address.